docs/pages/docs/getting-started.mdx

```diff
@@ -88,17 +88,17 @@ Add [`turbo`](https://www.npmjs.com/package/turbo) as a development dependency a
 <Tabs items={['npm', 'yarn', 'pnpm']} storageKey="selected-pkg-manager">
   <Tab>
     ```bash
-    npm install turbo -D
+    npm install turbo --save-dev
     ```
   </Tab>
   <Tab>
     ```bash
-    yarn add turbo -DW
+    yarn add turbo --dev --ignore-workspace-root-check
     ```
   </Tab>
   <Tab>
     ```bash
-    pnpm add turbo -Dw
+    yarn add turbo --save-dev --ignore-workspace-root-check
```

Comment:
Yarn in the Pnpm tab?

Reply:
good catch https://github.com/vercel/turborepo/pull/1775